core/unit_test/TestPolicyConstruction.hpp

```diff
@@ -795,7 +795,7 @@ TEST(TEST_CATEGORY, desired_occupancy_empty_base_optimization) {
   static_assert(sizeof(decltype(policy)) == 1, "");
   static_assert_dummy_policy_must_be_size_one<sizeof(decltype(policy))>
       _assert1{};
-  (void)_assert1;  // avoid unused variable warning
+  (void)&_assert1;  // avoid unused variable warning
```

Comment:
??

Reply:
`nvc++` still complained here using `(void)` but this syntax seems to silence the warning (on all CI checks).